Datamigratie.Server/Features/MigrateZaak/MigrateZaakService.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Datamigratie.Common.Services.Det.Models;
+using Datamigratie.Common.Services.OpenZaak;
+using Datamigratie.Common.Services.OpenZaak.Models;
+
+namespace Datamigratie.Server.Features.MigrateZaak
+{
+
+    public interface IMigrateZaakService
+    {
+        public Task<OzZaak> MigrateZaak(DetZaak detZaak, Guid ozZaaktypeId);
+    }
+
+    public class MigrateZaakService(IOpenZaakApiClient openZaakApiClient, IConfiguration configuration) : IMigrateZaakService
+    {
+        public async Task<OzZaak> MigrateZaak(DetZaak detZaak, Guid ozZaaktypeId)
+        {
+            CheckIfZaakAlreadyExists();
+
+            var createZaakRequest = CreateOzZaakCreationRequest(detZaak, ozZaaktypeId);
+            
+            var createdZaak = await openZaakApiClient.CreateZaak(createZaakRequest);
+
+            return createdZaak;
+        }
+
+        private static void CheckIfZaakAlreadyExists()
+        {
+            // TODO -> story DATA-48
+        }
+
+        private CreateOzZaakRequest CreateOzZaakCreationRequest(DetZaak detZaak, Guid ozZaaktypeId)
+        {
+            // First apply data transformation to follow OpenZaak constraints
+            var openZaakBaseUrl = configuration.GetValue<string>("OpenZaakApi:BaseUrl");
```

Comment:
Het is wel netter om de waarde niet hier rechtstreeks uit de configuration te halen, maar iets van services.AddOptions.... te doen bij het opstarten